src/plugins/intel_cpu/src/plugin.cpp

```diff
@@ -191,131 +190,6 @@ static bool streamsSet(const ov::AnyMap& config) {
-        }
-        return std::pair<std::string, StreamCfg>(std::to_string(streams_info.num_streams), streams_info);
-    };
-
-    auto getPerfHintName = [&]() {
-        const bool streamsExplicitlySetForModel = streamsSet(config);
-        // checking streams (to avoid overriding what user might explicitly set in the incoming config or previously via
-        // SetConfig)
-        if (streamsExplicitlySetForModel || streamsExplicitlySetForEngine)
-            return std::string();
-
-        const auto& perf_hint = config.find(ov::hint::performance_mode.name());
-        /* performance hints set for network has higher pririty than engine ones.
-         * This applies for all the configuration parameters */
-        const auto& perf_hint_name = (perf_hint != config.end()) ? perf_hint->second.as<std::string>()
-                                                                 : ov::util::to_string(engConfig.hintPerfMode);
-        return perf_hint_name;
-    };
-
-    // We compute both hints values because the optimal number of streams are computed based on ov::Model
-    // while we export model in cpu internal opset so we need to save precomputed optimal # streams for both hint modes
-    const auto latency_hints = ov::util::to_string(ov::streams::NUMA);
-    const auto tput_hints = getNumStreamsThroughput();
-
-    // save hints parameters to model rt_info
-    ov::AnyMap hints_props;
-    const auto latency_name =
-        ov::util::to_string(ov::hint::PerformanceMode::LATENCY) + "_" + std::string(ov::num_streams.name());
-    const auto tput_name =
-        ov::util::to_string(ov::hint::PerformanceMode::THROUGHPUT) + "_" + std::string(ov::num_streams.name());
-    hints_props.insert({latency_name, latency_hints});
-    hints_props.insert({tput_name, std::to_string(tput_hints.second.num_streams)});
-    model->set_rt_info(hints_props, "intel_cpu_hints_config");
-
-    const auto perf_hint_name = getPerfHintName();
-    if (perf_hint_name == ov::util::to_string(ov::hint::PerformanceMode::LATENCY)) {
-        OPENVINO_SUPPRESS_DEPRECATED_START
-        config[CONFIG_KEY(CPU_THROUGHPUT_STREAMS)] = CONFIG_VALUE(CPU_THROUGHPUT_NUMA);
-        OPENVINO_SUPPRESS_DEPRECATED_END
-        config[ov::num_streams.name()] = latency_hints;
-    } else if (perf_hint_name == ov::util::to_string(ov::hint::PerformanceMode::THROUGHPUT)) {
-        OPENVINO_SUPPRESS_DEPRECATED_START
-        config[CONFIG_KEY(CPU_THROUGHPUT_STREAMS)] = tput_hints.first;
-        OPENVINO_SUPPRESS_DEPRECATED_END
-        config[ov::num_streams.name()] = tput_hints.first;
-        config[ov::threading::big_core_streams.name()] = std::to_string(tput_hints.second.big_core_streams);
-        config[ov::threading::small_core_streams.name()] = std::to_string(tput_hints.second.small_core_streams);
-        config[ov::threading::threads_per_stream_big.name()] = std::to_string(tput_hints.second.threads_per_stream_big);
```

Comment:
Maybe we also need remove below unused stream_executor properties?
```
ov::threading::big_core_streams
ov::threading::small_core_streams
ov::threading::threads_per_stream_big
ov::threading::threads_per_stream_small
ov::threading::small_core_offset
```

Reply:
Removed now.